src/storage/src/hummock/store/version.rs

```diff
@@ -417,6 +417,13 @@ impl HummockVersionReader {
         let (imms, uncommitted_ssts, committed_version) = read_version_tuple;
         let min_epoch = gen_min_epoch(epoch, read_options.retention_seconds.as_ref());
 
+        if read_options.prefix_hint.is_some() {
```

Comment:
Should we move this to the line after `for imm in &imms` loop?

Reply:
Fix it , Thanks